cpp/ql/test/library-tests/dataflow/taint-tests/taint.cpp

```diff
@@ -85,14 +85,14 @@ void class_field_test() {
 
 	mc1.myMethod();
 
-	sink(mc1.a);
-	sink(mc1.b); // tainted [NOT DETECTED with IR]
-	sink(mc1.c); // tainted [NOT DETECTED with IR]
-	sink(mc1.d); // tainted [NOT DETECTED with IR]
-	sink(mc2.a);
-	sink(mc2.b); // tainted [NOT DETECTED with IR]
-	sink(mc2.c); // tainted [NOT DETECTED with IR]
-	sink(mc2.d);
+	sink(mc1.a); // [FALSE POSITIVE]
+	sink(mc1.b); // tainted
+	sink(mc1.c); // tainted
+	sink(mc1.d); // tainted
+	sink(mc2.a); // [FALSE POSITIVE]
+	sink(mc2.b); // tainted
+	sink(mc2.c); // tainted
+	sink(mc2.d); // [FALSE POSITIVE]
```

Comment:
I no longer see the `.expected` changes that correspond to these comment changes.

Reply:
You're right. I think these results were lost when we limited `PostUpdateNodes` to be on chi instructions with non-conflated results. I've fixed the annotations in 62e2ffe62346a01a4bf63eca9d0b747919f7738f.